qa/suites/rbd/nvmeof/base/install.yaml

```diff
@@ -0,0 +1,32 @@
+use_shaman: True
+tasks:
+- cephadm:
+- cephadm.shell:
+    host.a:
+    # get state before nvmeof deployment
+    - ceph orch status
+    - ceph orch ps
+    - ceph orch ls
+    - ceph orch host ls
+    - ceph orch device ls
+    - ceph osd lspools
+    # create pool
+    - ceph osd pool create mypool
+    - ceph osd pool application enable mypool rbd
```

Comment:
use the next command instead of osd pool application command
`rbd pool init -p mypool`